src/Advection/cell_advection_timescale.jl

```diff
@@ -12,8 +12,7 @@ end
     Δy = Δyᶜᶠᶜ(i, j, k, grid)
     Δz = Δzᶜᶜᶠ(i, j, k, grid)
 
-    return @inbounds min(Δx / abs(u[i, j, k]),
-                         Δy / abs(v[i, j, k]),
-                         Δz / abs(w[i, j, k])) 
+    return @inbounds 1 / (abs(u[i, j, k]) / Δx +
+                          abs(v[i, j, k]) / Δy +
+                          abs(w[i, j, k]) / Δz) 
```

Comment:
is this the proper way to compute CFL? I'll trust you on that... That's like the geometric mean of the timescales in each direction?

Reply:
Imagine a 2D situation, we start from
$$\frac{\partial c}{\partial t} = u\frac{\partial c}{\partial x} + v\frac{\partial c}{\partial y}$$
We have advection in x and y, then we can approximate the CFL condition as
$$|u|\frac{\Delta t}{\Delta x} + |v|\frac{\Delta t}{\Delta y} < CFL (\text{ maximum 1 })$$
This means that 
$$\Delta t / \tau < CFL$$
where 
$$\tau = \left(\frac{|u|}{\Delta x} + \frac{|v|}{\Delta y} \right)^{-1}$$
The extension to 3D includes also the advection in z

Reply:
Sounds good! Why don't we include all this in the docstring of the `cell_advection`?